backend/requirements.txt

```diff
@@ -1,11 +1,74 @@
+aniso8601==7.0.0
+appdirs==1.4.4
+asgiref==3.3.1
+backcall==0.2.0
+black==20.8b1
+certifi==2020.12.5
+cffi==1.14.4
+chardet==4.0.0
+click==7.1.2
+cryptography==3.3.1
+dataclasses==0.8
+decorator==4.4.2
 Django==3.1.5
+django-anymail==8.2
+django-cors-headers==3.7.0
 django-extensions==3.1.0
-jedi<0.18.0
-ipython==7.19.0
-psycopg2==2.8.6
-python-decouple==3.4
+django-filter==2.4.0
+django-graphql-auth==0.3.15
+django-graphql-jwt==0.3.0
+docker==4.4.1
+english==2020.7.0
+environs==9.3.0
+et-xmlfile==1.0.1
+Faker==5.6.5
+graphene==2.1.8
+graphene-django==2.15.0
+graphql-core==2.3.2
+graphql-relay==2.0.1
+idna==2.10
+ipython==7.16.1
+ipython-genutils==0.2.0
+jdcal==1.4.1
+jedi==0.17.2
+marshmallow==3.10.0
+monotonic==1.5
+mypy-extensions==0.4.3
+neotime==1.7.4
+numpy==1.19.5
 openpyxl==3.0.6
+packaging==20.8
+pandas==1.1.5
+pansi==2020.7.3
+parso==0.7.1
+pathspec==0.8.1
+pexpect==4.8.0
+pickleshare==0.7.5
+promise==2.3
+prompt-toolkit==2.0.10
+psycopg2==2.8.6
+ptyprocess==0.7.0
 py2neo==2020.1.1
-graphene_django==2.15.0
-django-cors-headers==3.7.0
-django-anymail[sendgrid]==8.2
\ No newline at end of file
+pycparser==2.20
+Pygments==2.7.4
+PyJWT==1.7.1
+pyparsing==2.4.7
+python-dateutil==2.8.1
+python-decouple==3.4
+python-dotenv==0.15.0
+pytz==2020.5
+regex==2020.11.13
+requests==2.25.1
+Rx==1.6.1
+singledispatch==3.4.0.3
+six==1.15.0
+sqlparse==0.4.1
+text-unidecode==1.3
+toml==0.10.2
+traitlets==4.3.3
+typed-ast==1.4.2
+typing-extensions==3.7.4.3
+urllib3==1.26.2
+wcwidth==0.2.5
+websocket-client==0.57.0
+xlrd==2.0.1
```

Comment:
Davide, uprav prosim requirements tak, aby obsahoval jen balicky ktere potrebujem. Vypada to, ze jsi do nej dal i vsechny souvisejici zavislosti, coz nechceme (az budeme v budoucnu projekt upgradovat, nebude jasne co je hard requirements definovany nami a co vynucena zavislost z jinych balicku)

Pro srovnani -- v master vetvi je ted v requirement 10 zavislosti, v tvem PR 74.